cts/query/where.json

```diff
@@ -0,0 +1,47 @@
+{
+  "number": {
+    "request": {
+      "find": "values",
+      "filter": {
+        "$where": "this.v == 42"
```

Comment:
We don't support `$where` operator yet. As we don't have any expected failure mechanism (and probably don't plan to add it), should we keep this test here, or could we remove it for now to make working on further cts tests easier, and reintroduce it later?

Reply:
https://github.com/OpenDocDB/cts/issues/54